stable/spring-cloud-data-flow/Chart.yaml

```diff
@@ -1,17 +1,13 @@
 apiVersion: v1
-description: Toolkit for building data processing pipelines.
+description: DEPRECATED Toolkit for building data processing pipelines.
 name: spring-cloud-data-flow
-version: 2.7.1
+version: 2.7.2
 appVersion: 2.5.1.RELEASE
+# The redis chart is deprecated and no longer maintained. For details deprecation, see the PROCESSES.md file.
```

Comment:
I'm guessing that this was a copy and paste error. Did you mean to say redis?

Reply:
Yeah, fixed.